docs/tutorials/04-writing-shaders.md

```diff
@@ -67,6 +66,16 @@ WireframePass::WireframePass()
     mpGraphicsState->setRasterizerState(mpRasterState);
 }
 ```
+## `reflect()`
+As in the Implementing a Render Pass tutorial, you simply set the Output for the Wireframe view.
+```
+RenderPassReflection WireframePass::reflect(const CompileData &compileData)
```

Comment:
Falcor's coding conventions uses pointer associated with the type, i.e. `const CompiledData& compiledData`